src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -345,5 +349,673 @@ public static bool IsLeftSideOfCompoundAssignExpression(this SyntaxNode node)
                 node.Parent.IsCompoundAssignExpression() &&
                 ((AssignmentExpressionSyntax)node.Parent).Left == node;
         }
+
+        /// <summary>
+        /// Returns the list of using directives that affect <paramref name="node"/>. The list will be returned in
+        /// top down order.  
+        /// </summary>
+        public static IEnumerable<UsingDirectiveSyntax> GetEnclosingUsingDirectives(this SyntaxNode node)
```

Comment:
All existing code just moved down to C# compiler extensions shared project.

Reply:
ok.  and these are not the context query extensions anymore?

Reply:
Yes, all the extensions file within the ContextQuery sub-folders have been reverted to original. These are just general extension methods being moved down.